src/snackbar/index.tsx

```diff
@@ -19,40 +18,34 @@ export interface SnackbarProperties {
 	stacked?: boolean;
 }
 
-@theme(css)
-export class Snackbar extends ThemedMixin(WidgetBase)<SnackbarProperties> {
-	protected render(): DNode {
-		const { type, open, leading, stacked, messageRenderer, actionsRenderer } = this.properties;
+const factory = create({ coreTheme: theme }).properties<SnackbarProperties>();
```

Comment:
why is this `coreTheme` not `theme`? seems odd

Reply:
Ah, this was just a side-effect of the way I refactored it. I went ahead and updated it!